### PR TITLE
Add social login using Github

### PR DESCRIPTION
**Pull request by alaa-yahia**

### CODE REVIEW

File: src/pages/connect/github.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { handleAuthentication } from '../../services/auth'
+
+const Github = () => {
+  const url = process.env.GATSBY_STRAPI_API_URL || 'http://localhost:8082'
+  const { jwt } = handleAuthentication(url, 'github')
```
Comment (kabaros) [inline src/pages/connect/github.js:6]: I don't think you need to destructure `{jwt}`? it's not used anywhere

File: src/components/Signin/SigninForm.js [hunk 2]
```diff
@@ -104,6 +104,16 @@ const SigninForm = () => {
                   Forgot Password?
                 </a>
               </div>
+              <div className="text-center">
+                <h1 className="font-bold text-grey-darker block mb-2 mt-5">
+                  Or
+                </h1>
+                <button className="py-2 px-4 bg-gray-700 text-white text-center rounded hover:bg-gray-600 focus:outline-none mt-5 ml-4">
+                  <a href="http://localhost:8082/connect/github">
```
Comment (kabaros) [inline src/components/Signin/SigninForm.js:112]: this should be the strapi api .. not local

File: src/pages/connect/github.js (hunk 1 above)
Comment (alaa-yahia) [reply src/pages/connect/github.js:6]: Yeah, actually no need to return anything from `handleAuthentication`

**Comment by alaa-yahia** [comment]
@abolkog @kabaros It is not working on the live website, when clicking on the github button it will redirect me to `https://edu.nyaladev.com/connect/github/` without opening github sign in page

**Comment by kabaros** [comment]
I think maybe we need to setup GATSBY_STRAPI_URL on the build script
@abolkog ?

**Pull request merged**